### ISSUE TITLE
Bug: "Run on boot" not working

### ISSUE THREAD

**Issue by naamaloomi** [Kyusung4698/PoE-Overlay#419]
Installed PoE-Overlay using the setup file. PoE-Overlay shows up in the list of Startup Apps, but does not start with Windows.
Running on Windows 10 Home.
Not sure what information is relevant here. If you need more, let me know.

**Comment by Lokeras** [comment]
i confirm that i have the same issue, also win10 on 2 different pc. I checked settings run on boot is on, checked task managed, app is enabled in start up.

**Comment by Kyusung4698** [comment]
Hi, thanks for creating an issue. Could you please check the Event Log? Maybe there is a error entry. I'll look into it.

**Comment by naamaloomi** [comment]
I couldn't find any error that seemed relevant in the event viewer (the only error I could find was about svchost failing to open a corrupt log file).

In the PoE-Overlay log I couldn't find any errors, so I deleted it and rebooted my computer. After the reboot, PoE-Overlay did not start and no new log was created. After manually starting PoE-Overlay, the log showed up again but still no errors in it.

This seems to point towards Windows not allowing PoE-Overlay to start, but I would've expected that to be logged in the event viewer.

I also uninstalled and re-installed PoE-Overlay, but that made no difference. The only things in the PoE-Overlay log are these (I replaced the actual path with %appdata%, because paranoia):
```[2020-03-21 13:34:45.117] [info] App starting...
[2020-03-21 13:34:45.125] [info] App checking for animation flag: %appdata%\poe-overlay\animation.flag.
[2020-03-21 13:34:46.734] [info] App Version:  0.6.9
[2020-03-21 13:34:46.880] [info] Checking for update
[2020-03-21 13:34:47.829] [info] Update for version 0.6.9 is not available (latest version: 0.6.9, downgrade is disallowed).
[2020-03-21 13:35:23.306] [info] set bounds to:  { x: 0, y: 0, width: 3440, height: 1440 }
```

**Comment by Kyusung4698** [comment]
Thanks for your update. I think i know the reason. I've introduced requestExecutionLevel highest with version 0.6.5. This requires the app to be run with the highest available permissions.

I'm using [node-auto-launch](https://github.com/Teamwork/node-auto-launch) to add the auto start entry and there is an open issue about running an app as highest available on startup. See:
https://github.com/Teamwork/node-auto-launch/issues/86
Unfortunately this issue has not been fixed but i think i found a solution on stackoverflow:
https://stackoverflow.com/questions/9544454/regedit-to-run-an-exe-at-startup-as-administrator

Now i just need to create a PR at this repo and fix it. Then it should work.

**Comment by Kyusung4698** [comment]
## 0.6.10 (2020-03-22)

- fix run on boot not working (#419)

You may need to reenable the setting and remove a duplicate of the startup menu.

**Comment by naamaloomi** [comment]
I can confirm that it's working now.
